cvmfs/cvmfs.cc

```diff
@@ -1580,11 +1577,11 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
     download_manager_->GetTimeout(&seconds, &seconds_direct);
     attribute_value = StringifyInt(seconds_direct);
   } else if (attr == "user.rx") {
-    int64_t rx = uint64_t(download_manager_->GetStatistics().transferred_bytes);
+    int64_t rx = download_manager_->GetCounters().no_transferred_bytes->Get();
```

Comment:
This can be replaced by `Statistics->Lookup("n_transferred_bytes")`, in order to get rid of the `GetCounters()` method.